commands/manifest/template.go

```diff
@@ -118,3 +109,50 @@ func templateCmd(c *cli.Context) error {
 
 	return nil
 }
+
+func GetTemplatesManifests(m dx.Manifest) (string, error) {
+
+	switch {
```

Comment:
This switch statement is verbose. Accounts for all cases, but can be simplified a lot:

- if `m.Chart.Name != ""` we should template a Helm chart.
- If manifests are there, we should always just add at the end of the list.

We don't need a distinct case for both conditions are true at the same time.

- So first, define a var that holds all manifests,
- then add the templated helm chart if it is provided
- then add the raw manifests if it is provided

Will be a lot shorter.